src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -1306,7 +1318,16 @@ public static class GroupRowFormat
             public static string Simulation = "Simulation";
             public static string Time = "Time";
             public static string Observation = "Observation";
-            public static string Undefined = "Undefined";
+            public static string IdentityCurve = "Identity";
+            public static string Undefined = "Undefined"; 
```

Comment:
Let's not use undefined for this stuff. Did not we say we were going to create a special group for this?

Reply:
The new special group is Identity. Undefined was there already - it is being shown as deleted and then added because Identity got added before it (for me it also makes sense to leave the Undefined for last, even in the constants).